lib/vector-core/src/tls/settings.rs

```diff
@@ -109,12 +109,14 @@ pub struct TlsConfig {
     ///
     /// Declare the supported ALPN protocols, which are used during negotiation with peer. Prioritized in the order
     /// they are defined.
+    #[configurable(metadata(docs::examples = "[\"h2\"]"))]
```

Comment:
I _think_ you'd just want this to be `docs::examples = "h2"`? Since the field is a vector, it should show the examples natively being passed as an array, even if you add them here as individual items.

I _think_.

Reply:
Yeah there's still some magic happening separate from here where Array examples get inserted in as items in the Array, same for objects, etc.

We can double check after this merges and we use it on a user facing page, but I think you're right and I'll just update it here.

Reply:
Updated in [073aac8](https://github.com/vectordotdev/vector/pull/15658/commits/073aac888bb89dbaed24ad4a8684f796d255f8c8)